reflection.md

```diff
@@ -2,8 +2,33 @@
 
 ## Which operators were easiest to implement and why?
 
+All of the operators were easier to implement than expected.  Scala seems
+to provide the coder with lots of freedom for choosing operators to define or
+overload.  I would say that the operators ||, ~, and <*> were the easiest to
```

Comment:
I definitely agree that these were rather easy to implement since they were direct translations.